### ISSUE TITLE
Loading spinner stays on error

### ISSUE THREAD

**Issue by xeruf** [plepe/OpenStreetBrowser#158]
It keeps on retrying, despite being rejected with "Too many requests"

<img width="923" alt="image" src="https://user-images.githubusercontent.com/13354331/193809645-6d2fe043-e9a8-4d67-8a17-c85506ce4af0.png">

**Comment by plepe** [comment]
Yes that's right. It will retry loading up to three times (which is currently hard-coded). Between retry requests, there will be a pause of increasing time.

The relevant code is in module [overpass-frontend](https://github.com/plepe/overpass-frontend/). You can configure the options with the parameter 'overpassOptions' in OpenStreetBrowser's conf.php.

Here are the relevant options in overpass-frontend: https://github.com/plepe/overpass-frontend/blob/master/src/OverpassFrontend.js#L81

**Comment by xeruf** [comment]
maybe replace the spinner by red text with the error code?
